dallinger/command_line/docker_ssh.py

```
@@ -300,20 +324,24 @@ def deploy(
         "dallinger/Caddyfile",
     )
```

Can we also add a check that throws a sensible error if the app doesn't exist already?

---

Sure! I added a check to fail early in that case.